src/components/Match/matchPages.jsx

```diff
@@ -45,45 +45,43 @@ const filterMatchPlayers = (players, team = '') =>
     ((team === 'radiant' && isRadiant(player.player_slot)) || (team === 'dire' && !isRadiant(player.player_slot)) || team === '')
   ).sort((a, b) => a.player_slot - b.player_slot);
 
-const MatchPlayersTable = ({
-  match,
-  columns,
-}) => (<Table data={match.players} columns={columns} />);
-
-const MatchPlayersTableSplit = ({
+const TeamTable = ({
   match,
   columns,
+  radiant,
+  dire,
 }) => (<div>
+  <Heading title={radiant} />
   <Table data={filterMatchPlayers(match.players, 'radiant')} columns={columns} />
+  <Heading title={dire} />
   <Table data={filterMatchPlayers(match.players, 'dire')} columns={columns} />
 </div>);
 
+// TODO: Add networth and xp graph to overview
 const matchPages = [{
   name: strings.tab_overview,
   content: match => (<div>
-    <Heading title={strings.heading_overview} />
-    <MatchPlayersTableSplit match={match} columns={overviewColumns(match)} />
-    <Heading title={strings.heading_ability_builds} />
-    <MatchPlayersTable match={match} columns={abilityUpgradeColumns} />
+    <TeamTable match={match} columns={overviewColumns(match)} radiant={strings.general_radiant} dire={strings.general_dire} />
```

Comment:
You shouldn't need to pass the radiant/dire strings as props if they aren't going to change.